src/views/my-account/my-account.view.jsx

```diff
@@ -150,6 +151,7 @@ const mapDispatchToProps = (dispatch) => ({
       type: 'page',
       data: {
         title: 'My Account',
+        subtitle: packagejsonVersion,
```

Comment:
Maybe this should be now a template string: `version ${packagejsonVersion}`?

Reply:
Why not :)